firmware/arduino/chessboard/motorDriver.ino

```diff
@@ -74,11 +74,11 @@ void disableMotors()
 // Drives the motor corresponding to "motor" to be aligned properly at either the max position or 0
 void alignAxis(uint8_t motor[], uint8_t alignmentCode)
 {
-  int *currentMotorPos;
-  int eighthStepsPerPulse;
-  int i;
+  uint16_t *currentMotorPos;
```

Comment:
I think this variable name should be updated to reflect that it's a pointer. Maybe `currentMotorPosPtr`